packages/calcite-components/src/components/flow/flow.tsx

```diff
@@ -1,6 +1,6 @@
-import { Component, Element, h, Listen, Method, State, VNode } from "@stencil/core";
+import { Component, Element, h, Listen, Method, Prop, State, VNode } from "@stencil/core";
 import { createObserver } from "../../utils/observers";
-import { FlowDirection } from "./interfaces";
+import { FlowDirection, FlowItemLikeElement } from "./interfaces";
```

Comment:
I think `FlowItemLikeElement` is fine but it could also be `CustomFlowItemElement`? I'm fine either way.

Reply:
I like your name suggestion better, but the interface only requires the props expected by the parent and not all from `flow-item`.